.pre-commit-hooks.yaml

```diff
@@ -56,3 +64,10 @@
   files: poetry.lock
   require_serial: true
   pass_filenames: false
+
+- id: merge-coverage-report
+  name: merge-coverage-report
+  entry: merge-coverage-report
+  language: python
+  require_serial: true
+  pass_filenames: false
```

Comment:
What do those keys stand for?

Reply:
Here is a [link](https://pre-commit.com/) to the pre-commit documentation.
The key pass_filenames is in order that the coverage-analyze will run only once in each pre-commit run, instead of for each file.
The key require_serial - if true this hook will execute using a single process instead of in parallel.